### ISSUE TITLE
Update to pytorch 0.4?

### ISSUE THREAD

**Issue by xiaoguo1995** [Spandan-Madan/A-Collection-of-important-tasks-in-pytorch#2]
Hi Spandan,

thank you very much for this pytorch quick tutorial. I find this link from torch forum when I want to define my custom loss function.

Could you please consider to update this excellent tutorial to pytorch 0.4 so that some tricky parts like "variable" or something could be more clear to new users. 

thank you very much again!

Best,
Peter

**Comment by Spandan-Madan** [comment]
Hi @PeterXiaoGuo,

I guess I am waiting for someone to fork, update and add a pull request with the updated version. Want to contribute? :)

**Comment by oneTaken** [comment]
@PeterXiaoGuo @Spandan-Madan
 If it's necessary, I would update this tutorial to pytorch **0.4**.  By the way, is the **index.html**  transformed from a notebook?

**Comment by Spandan-Madan** [comment]
Hi @oneTaken sounds great! I think it's a nice idea but given some research commitments I don't have the time to take this task up. If you can, please do update the code to 0.4! I can take care of the html file if you update the jupyter notebook! :)

**Comment by mugheesahmad** [comment]
hi, 

if its still open, I can try to contribute to make it compatible to latest.
Thanks.
